app/components/primer/beta/button.pcss

```diff
@@ -329,4 +329,19 @@ summary.Button {
   &.Button--large {
     width: var(--primer-control-large-size, 40px);
   }
+
+  /* Increase touch target */
+  @media (pointer: coarse) {
```

Comment:
@jonrohan I think this is already being accomplished at line 32 with a mixin. You can change the values to 44px instead 😄